index.html

```diff
@@ -2,358 +2,365 @@
+    <div class="popup-window hidePopup"></div>
+    <div class="project">
+      <h2>Projects</h2>
+      <img class="dots" src="./images/image_dots2.png" alt="">
+      <img class="geo4" src="./images/image-geometry_4.png" alt="">
+      <img class="geo5" src="./images/image-geometry_5.png" alt="">
+      <img class="geo6" src="./images/image-geometry_6.png" alt="">
+      <img class="geo7" src="./images/image-geometry_7.png" alt="">
+      <img class="teal" src="./images/image_dots_with_teal.png" alt="">
+      <figure>
+        <img class="pc" src="./images/pcprogect.png" alt="Project1">
+        <div class="box01">
+          <p>Project name goes here</p>
+          <ul>
+            <li><button class="html" type="button">HTML/CSS</button></li>
+            <li><button class="ruby" type="button">Ruby on Rails</button></li>
+            <li><button class="js" type="button">JavaScript</button></li>
+          </ul>
+          <div class="pjbutton">
+            <a class="actbutton">See this project</a>
+            <img class="narrow" src="./images/Union.png" alt="ico_narrow">
+          </div>
         </div>
-        <div class="project">
-            <figure>
-                <img class="pc1" src="./images/pcprogect1.png" alt="Project2">
-                <div class="box04">
-                    <p>Project name goes...</p>
-                    <ul>
-                        <li><button class="html" type="button">HTML/CSS</button></li>
-                        <li><button class="ruby" type="button">Ruby on Rails</button></li>
-                        <li><button class="js" type="button">+1</button></li>
-                    </ul>
-                    <div class="pjbutton">
-                        <a class="actbutton" href="project1.html">See this project</a>
-                        <img class="narrow" src="./images/Union.png" alt="ico_narrow">
-                    </div>
-                </div>
-            </figure>
-            <div class="box">
-                <p>Project name goes here</p>
-                <ul>
-                    <li><button class="html" type="button">HTML/CSS</button></li>
-                    <li><button class="ruby" type="button">Ruby on Rails</button></li>
-                    <li><button class="js" type="button">JavaScript</button></li>
-                </ul>
-                <div class="pjbutton">
-                    <a class="actbutton" href="project2.html">See this project</a>
-                    <img class="narrow" src="./images/narrow.png" alt="follow">
-                </div>
-            </div>
+      </figure>
+      <div class="box">
+        <p>Project name goes here</p>
+        <ul>
+          <li><button class="html" type="button">HTML/CSS</button></li>
+          <li><button class="ruby" type="button">Ruby on Rails</button></li>
+          <li><button class="js" type="button">JavaScript</button></li>
+        </ul>
+        <div class="pjbutton">
+          <a class="actbutton">See this project</a>
```

Comment:
- [ ] As you can see in the screenshots below the buttons in your Modal are not centered as specified in the original design, and the `See source` button **background-color** property should be `#1A2236` not white, Kindly be sure to respect the provided design 😃 

Current design            |  original design
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/163411575-82b174f0-4129-4738-9258-c258057f790c.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/163411464-ccf36a2b-7416-4865-8514-7a4fa1653255.PNG)